src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/configuration/configuration.component.ts

```diff
@@ -110,25 +100,7 @@ export class ConfigurationComponent implements OnInit {
         cellClass: 'wrap',
         cellTemplate: this.confValTpl
       },
-      { flexGrow: 1, prop: 'source' },
-      { flexGrow: 2, prop: 'desc', name: 'Description', cellClass: 'wrap' },
-      { flexGrow: 2, prop: 'long_desc', name: 'Long description', cellClass: 'wrap' },
-      {
-        flexGrow: 2,
-        prop: 'flags',
-        name: 'Flags',
-        cellClass: 'wrap',
-        cellTemplate: this.confFlagTpl
-      },
-      { flexGrow: 1, prop: 'type' },
-      { flexGrow: 1, prop: 'level' },
-      { flexGrow: 1, prop: 'default', cellClass: 'wrap' },
-      { flexGrow: 2, prop: 'daemon_default', name: 'Daemon default' },
-      { flexGrow: 1, prop: 'tags', name: 'Tags' },
-      { flexGrow: 1, prop: 'services', name: 'Services' },
-      { flexGrow: 1, prop: 'see_also', name: 'See_also', cellClass: 'wrap' },
-      { flexGrow: 1, prop: 'max', name: 'Max' },
-      { flexGrow: 1, prop: 'min', name: 'Min' }
+      { flexGrow: 1, prop: 'default', cellClass: 'wrap' }
```

Comment:
IMO, both `default` and `current value`  columns should have the same `flexGrow` (or you can simply remove the `flexGrow` from all columns).

Reply:
Okay, will do.